contracts/src/systems/trade/contracts/trade_systems.cairo

```diff
@@ -314,17 +316,20 @@ mod trade_systems {
                 ));
             };
 
+            let maker_realm = get!(world, trade.maker_id, Realm );
+            let taker_realm = get!(world, trade.taker_id, Realm );
 
             emit!(world, OrderAccepted {
                 trade_id, 
-                maker_id: trade.maker_id,
-                taker_id, 
-                maker_resources: resource_chest::get_contents(world, trade.maker_resource_chest_id),
-                taker_resources: resource_chest::get_contents(world, trade.taker_resource_chest_id),
+                maker_realm_entity_id: maker_realm.entity_id,
+                maker_realm_id: maker_realm.realm_id,
+                taker_realm_entity_id: taker_realm.entity_id,
+                taker_realm_id: taker_realm.realm_id,
+                    maker_resources: resource_chest::get_contents(world, trade.taker_resource_chest_id),
```

Comment:
Sanity check, maker takes the content of the chest created by taker and vice versa. Correct ?

Reply:
Yes exactly, we want to have the resources that the maker/taker is giving. But the content of the chests are the resources that each of them is getting